source/Cosmos.IL2CPU/ILOpCodes/OpNone.cs

```diff
@@ -142,6 +142,7 @@ public override int GetNumberOfStackPops(MethodBase aMethod)
         case Code.Ldelem_U2:
         case Code.Ldelem_U4:
           return 2;
+        case Code.Ldelem_R4:

```

Comment:
`case Code.Ldelem_R4:` as well as `case Code.Ldelem_R8:` should be in the line above `return 2;`